recipes/alps/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "alps" %}
+{% set version = "2.3.b1" %}
+{% set revision = "r7782" %}
+{% set sha256 = "69910b4fcd5e98a01741843de530f896864a390f693bf010b2db56ceb3f071d7" %}
+{% set blas = os.environ.get('BLAS') or 'openblas' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-{{revision}}-src.tar.gz
+  url: http://alps.comp-phys.org/static/software/releases/{{ name }}-{{ version }}-{{revision}}-src.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - no_pyplot_in_test.patch
+    - no_install_deplibs.patch  # [win]
+    - boost_defs.patch  # [win]
+    - hdf5_win.patch  # [win]
+    - mps_win.patch  # [win]
+    - r7818.patch  # [win]
+    - cstdint_types.patch  # [win]
+    - parapack_win.patch  # [win]
+    - win_test_outputs.patch  # [win]
+
+build:
+  skip: true  # [not py27]
+  skip: true  # [np<111]
```

Comment:
So this selector seems to be causing issues during feedstock conversion. Not sure if `<` works, but we know `==`/`!=` works with NumPy. Also having 3 different `skip`s potentially may be a parsing issue. These can all be `or`ed together. 

ref: https://travis-ci.org/conda-forge/staged-recipes/builds/190627621#L230-L233
ref: https://github.com/conda-forge/scipy-feedstock/blob/e3e4b9778c06ecc0249f0c0ee607682b5efbdba6/recipe/meta.yaml#L18